packages/graphql-mesh-server/lib/fargate.ts

```diff
@@ -176,6 +181,7 @@ export class MeshService extends Construct {
 
     const cluster = new ecs.Cluster(this, `cluster`, {
       vpc: this.vpc,
+      containerInsights: props.containerInsights || true,
```

Comment:
This will always evaluate to true, you have to do:
```
containerinsights: props.containerInsights !== undefined ?  props.containerInsights :  true,
```

Otherwise you can never turn it off

Reply:
Oh, of course :disappointed:

Reply:
Fixed :slightly_smiling_face: